.gitignore

```diff
@@ -16,3 +16,5 @@ NewVisualizingRussian*
 /scratch
 /django_cache
 /russian.csv
+parser_tool/static/js/src/2-26-ANN-test.ann
+parser_tool/static/js/src/ANNOY_tree.ann
```

Comment:
Can you add a brief comment to the gitignore explaining why this is being ignored (e.g. it's a 1GB index that can be generated from the navec vector file)? Mostly for future reference when someone is looking at this and wondering what it is and why it's ignored. 

Perfect. Since this will be created at build/deploy time it should be ignored in the repository. As discussed, however, we are storing the navec vector file in the git repository since it's